app/views/admin/menu/_menu.html.erb

```diff
@@ -22,3 +22,6 @@
   <%= render partial: 'admin/menu/menu_item', locals:
       { text: 'Settings', link: admin_settings_path, icon: 'settings' } %>
 </ul>
+<script>
+  persist_menu_state();
```

Comment:
Should `persist_menu_state()`'s work be delayed until after the DOM has finished loading? 

It looks like the function searches the document  immediately with `document.getElementsByClassName( 'c-sidebar-nav-dropdown' )` which could be called before the DOM finishes loading and so miss some elements.

Reply:
Probably, yeah. I'm not very happy about anything in the recent JavaScript commits :expressionless:  So long as it works for me then I'm inclined to ignore it for now, and revisit it when I have time to learn more and redo it properly - or wait until somebody else downloads the project and fixes it or files an issue :smiley: